contracts/operators/StakeDAO/StakeDaoStrategyStorage.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.11;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/// @dev A Curve pool with its number of coins
+/// @param poolAddress The address of the curve pool
```

Comment:
We should standardize how we write `curve pool` in comments below we use `CurvePool `